ons_alpha/release_calendar/models.py

```diff
@@ -113,19 +120,18 @@ def status_label(self) -> str:
         return ReleaseStatus[self.status].label
 
     def get_template(self, request, *args, **kwargs):
-        if self.status == ReleaseStatus.UPCOMING:
-            return "templates/pages/release_page--upcoming.html"
+        if self.status == ReleaseStatus.PROVISIONAL:
```

Comment:
updated so as to be closer to the life site: https://www.ons.gov.uk/releases/privaterentandhousepricesukoctober2025 vs https://www.ons.gov.uk/releases/publicsectorfinancesukoctober2025timeseries